server/raft_transport.go

```diff
@@ -32,8 +33,19 @@ import (
 const (
 	raftServiceName = "MultiRaft"
 	raftMessageName = raftServiceName + ".RaftMessage"
+	// Outgoing messages are queued on a per-node basis on a channel of
+	// this size.
+	raftSendBufferSize = 500
+	// When no message has been sent to a Node for that duration, the
+	// corresponding instance of processQueue will shut down.
+	raftIdleTimeout = time.Minute
```

Comment:
I just didn't want goroutines for nodes which are removed to stay around. And firing up a goroutine with each request (we're doing fire-and-forget, so we might be faster than MultiRaft is feeding our channel) seemed wasteful as well. Operating on the premise that we will always drop the queue on the floor (which might be a good idea anyways, a lot of redundant heartbeats and appends piling up) I don't think this is an issue.
For testing your suggestion to shut down the goroutine eagerly (but then we should probably do it every time) makes sense but in production that seems like something to avoid.